src-tool/Pact/Analyze/Eval/Core.hs

```diff
@@ -312,6 +314,17 @@ evalCore (ListHash ty' xs) = do
       SList t' -> Pact.PList . V.fromList <$> traverse (reify t') c
       _ -> throwErrorNoLoc (FailureMessage "Unsupported type, currently we support integer, decimal, string, and bool")
 
+evalCore (Keccak256Hash xs) = eval xs <&> unliteralS >>= \case
+    Nothing ->do
```

Comment:
```suggestion
    Nothing -> do
```

Reply:
Also, do we want to have this branch explicitly, or what do you think of just computing the hash of an empty list (which is what `Nothing` implies here, I guess) to avoid hardcoding `h`?